src/CMakeLists.txt

```diff
@@ -12,9 +12,8 @@ IF(SUFFIX_SO_VERSION)
   SET_TARGET_PROPERTIES(${NAME} PROPERTIES SOVERSION ${PROJECT_VERSION})
 ENDIF(SUFFIX_SO_VERSION)
 
-SET(PYTHON_MODULE dynamic_graph/sot/tiago)
-PYTHON_INSTALL_ON_SITE("${PYTHON_MODULE}" "__init__.py" )
-PYTHON_INSTALL_ON_SITE("${PYTHON_MODULE}" "tiago.py" )
+TARGET_INCLUDE_DIRECTORIES(${NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
This comes from @nim65s changes and appear here because of the merge commit. Anyway, I think the deletion is only to remove duplication in the file. They are still installed.